op-node/rollup/engine/events.go

```diff
@@ -323,6 +325,29 @@ func (d *EngDeriver) OnEvent(ev event.Event) bool {
 			}
 		} else {
 			d.log.Info("successfully processed payload", "ref", ref, "txs", len(x.Envelope.ExecutionPayload.Transactions))
+			payload := x.Envelope.ExecutionPayload
+			if payload != nil {
+				latestBlockTimeStamp := uint64(payload.Timestamp)
+
+				currentTime := uint64(time.Now().Unix())
+
+				if latestBlockTimeStamp <= currentTime {
+					timeDiff := currentTime - latestBlockTimeStamp
+
+					if timeDiff < blockTimeThreshold {
```

Comment:
I wonder if we need to be more strict here, if I follow this would count the following scenario as healthy:

1. We have two second block times
2. The latest blocks time is 8 seconds
3. The current time is 9.9 seconds

I wonder if we need to have a configurable threshold for what we deem as late? I would imagine anything > 250ms or something.